packages/near-fast-auth-signer/src/components/SignMultichain/SignMultichain.tsx

```diff
@@ -97,21 +98,16 @@ function SignMultichain() {
     feeProperties: TransactionFeeProperties
   ) => {
     try {
-      const isUserAuthenticated = await getAuthState();
-      if (isUserAuthenticated !== true) {
-        onError('You are not authenticated or there has been an indexer failure');
-      } else {
```

Comment:
is it ok to exclude `isUserAuthenticated` check?

Reply:
I moved it to here: https://github.com/near/fast-auth-signer/blob/db527505aabb414eb9443d389a979a20cd647872/packages/near-fast-auth-signer/src/components/SignMultichain/SignMultichain.tsx#L130

We need to be authenticated before estimate the gas for the function call.